src/Context/PantherContext.php

```diff
@@ -94,7 +92,7 @@ public function iWaitForElementToBeVisible(string $element): void
         $driver = $this->getDriver();
 
         try {
-            $driver->wait(3000, WebDriverExpectedCondition::visibilityOfAnyElementLocated(WebDriverBy::cssSelector($element)));
+            $driver->getWaitHelper()->waitFor($element);
```

Comment:
why don't you create `WaitHelper::waitForVisibility()` (as well as `waitForInvisibility()`)?

Reply:
Created 🙂